jaiarobotics/jaiabot
# feature/confirmation-for-sending-mission-out-of-range

If a user tries to run a mission containing a run that will finish outside of comms range of the hub (250 meters), a dialog box will pop up asking for confirmation before the run is sent. 

A warning will also show up if the user attempts to send a mission while the hub has no location data.

Testing:
Out of Comms Range:
- Create a mission with run(s) ending outside of comms range
- Ensure that a confirmation message appears for each run ending out of range with a bot assigned to it

No Hub Location Data:
- Change line 1255 of CommandControl  (`if (hub.location.lat === undefined || hub.location.lon === undefined)` in `runMissions()`) to `if (hublat === undefined || hublon === undefined)`
- Insert the following code just above this line
`let hublat = undefined`
`let hublon = undefined`

This should force the warning to come up.
